addons/advanced_throwing/XEH_postInit.sqf

```diff
@@ -3,6 +3,17 @@
 // Fired XEH
 [QGVAR(throwFiredXEH), FUNC(throwFiredXEH)] call CBA_fnc_addEventHandler;
 
+// make grenades spin
+["ace_throwableThrown", { // local event
+    params ["_unit", "_grenade"];
+
+    private _config = _grenade call CBA_fnc_getObjectConfig;
+    private _torque = vectorNormalized getArray (_config >> QGVAR(torqueDirection)) vectorMultiply
+        getNumber (_config >> QGVAR(torqueMagnitude));
```

Comment:
This works fine.
Has getNumber always worked on strings or was this something recent?
I thought I remember having to do some `call compile getText` to evaluate stuff like this.

Reply:
It worked like this since A2.